webpack.config.js

```diff
@@ -22,15 +22,15 @@ const EXPLORER_APP = {
   envVars: {
     EXPLORER_APP_DOMAIN_REGEX_DEV: '^protocol-explorer\\.dev|^localhost:\\d{2,5}|^pr\\d+--gpui\\.review',
     EXPLORER_APP_DOMAIN_REGEX_STAGING: '^protocol-explorer\\.staging',
-    EXPLORER_APP_DOMAIN_REGEX_PROD: '^gnosis-protocol\\.io',
-    EXPLORER_APP_DOMAIN_REGEX_BARN: '^barn\\.gnosis-protocol\\.io',
+    EXPLORER_APP_DOMAIN_REGEX_PROD: '^explorer\\.cow\\.fi',
+    EXPLORER_APP_DOMAIN_REGEX_BARN: '^barn\\.explorer\\.cow\\.fi',
```

Comment:
One more change, can you make the regexes accept both old and new domains?
E.g.: `^(explorer\\.cow\\.fi|gnosis-protocol\\.io)`

For the transition period would be better to have them both working

Reply:
Done! 

What I'm not sure is if **barn** will be available for `EXPLORER_APP_DOMAIN_REGEX_BARN`  like as `barn.explorer.cow.fi`

Reply:
Yes, barn is available for the explorer domain.
It just isn't yet for the api domains.